notebooks/official/workbench/pricing_optimization/pricing-optimization.ipynb

```diff
@@ -0,0 +1,2206 @@
+{
```

Comment:
<p>Split into 4 separate cells</p><p> </p><p>Move Dataset after Objective</p><p>Use template for Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>